src/ExportPostman.php

```diff
@@ -152,4 +142,21 @@ public function makeItem($route, $method, $routeHeaders)
             ],
         ];
     }
+
+    protected function initRoutes(string $filename): void
+    {
+        $this->routes = [
```

Comment:
Let's update this to make it more readable. 😁 
```suggestion
        $this->structure = [
```